presets/4.3/tune/fpvian_basher_freestyle.txt

```diff
@@ -0,0 +1,181 @@
+
+#$ INCLUDE: presets/4.3/tune/defaults.txt
+
+# Defaults end
+
+
+#PRESETS START
+
+set tpa_mode = PD
+set anti_gravity_gain = 4200
+set iterm_relax_cutoff = 10
+
+set simplified_i_gain = 135
+set simplified_pi_gain = 110
+set simplified_dmax_gain = 0
+set simplified_feedforward_gain = 140
+set simplified_pitch_pi_gain = 105
+simplified_tuning apply
+
+
+#$ OPTION BEGIN (CHECKED): Set Dynamic Idle to 30
+set dyn_idle_min_rpm = 30
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Increase Feedforward Jitter Reduction to 13
+set feedforward_jitter_factor = 13
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Disable Throttle Boost
+set throttle_boost = 0
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): Enable full Battery Sag Compensation
+set vbat_sag_compensation = 100
+#$ OPTION END
+
+
+
+#$ OPTION BEGIN (UNCHECKED): Dshot600
+set motor_pwm_protocol = DSHOT600
+set dshot_bidir = ON
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): Dshot300
+set motor_pwm_protocol = Dshot300
+set dshot_bidir = ON
+#$ OPTION END
+
+
+
+#$ OPTION BEGIN (UNCHECKED): Filtering for NOISY builds
+# Set filters to default
+
+#$ INCLUDE: presets/4.3/filters/defaults.txt
+
+# Defaults end
+
+# Filter presets begin
+
+set gyro_lpf1_static_hz = 0
+set dyn_notch_count = 3
+set dyn_notch_q = 500
+set simplified_gyro_filter_multiplier = 130
+set simplified_dterm_filter_multiplier = 130
+simplified_tuning apply
+#$ OPTION END
+
+
+
+#$ OPTION BEGIN (UNCHECKED): Filtering for AVERAGE builds
+# Set filters to default
+
+#$ INCLUDE: presets/4.3/filters/defaults.txt
+
+# Defaults end
+
+#Filter presets begin
+set gyro_lpf1_static_hz = 0
+set dyn_notch_count = 3
+set dyn_notch_q = 500
+set gyro_lpf1_dyn_min_hz = 0
+set gyro_lpf1_dyn_max_hz = 0
+set simplified_gyro_filter_multiplier = 130
+set rpm_filter_harmonics = 2
+set simplified_dterm_filter_multiplier = 130
+simplified_tuning apply
+#$ OPTION END
+
+
+
+#$ OPTION BEGIN (UNCHECKED): Filtering for CLEAN builds (recommended)
+# Set filters to default
+
+#$ INCLUDE: presets/4.3/filters/defaults.txt
+
+# Defaults end
+
+# Filter presets begin
+# It is recommended to use this option, if possible, for best overall performance.
+
+set gyro_lpf1_static_hz = 0
+set dyn_notch_count = 1
+set dyn_notch_q = 500
+set gyro_lpf1_dyn_min_hz = 0
+set gyro_lpf1_dyn_max_hz = 0
+set simplified_gyro_filter_multiplier = 200
+set rpm_filter_harmonics = 2
+set simplified_dterm_filter_multiplier = 130
+set dterm_lpf1_dyn_expo = 10
+simplified_tuning apply
+#$ OPTION END
+
+
+
+#$ OPTION BEGIN (UNCHECKED): Filtering for VERY CLEAN builds
+# Set filters to default
+
+#$ INCLUDE: presets/4.3/filters/defaults.txt
+
+# Defaults end
+
+# Filter presets begin
+# VERY CLEAN option is only recommended if props/motors are always kept in pristine condition
+
+set gyro_lpf1_static_hz = 0
+set dyn_notch_count = 1
+set dyn_notch_q = 500
+set gyro_lpf1_dyn_min_hz = 0
+set gyro_lpf1_dyn_max_hz = 0
+set simplified_gyro_filter_multiplier = 200
+set rpm_filter_harmonics = 2
```

Comment:
i think its worth turning ON bidirectional dshot inside the option if this option requires bidirectional dshot?
And same for other filtering options, if you assume bdir is on there

Reply:
Moved enabling bidirectional dshot to inside each of the filtering options. Removed from DSHOT options.